{"bugs": {"633064": {"comments": [{"is_private": false, "tags": [], "creation_time": "2011-02-10T02:21:54Z", "bug_id": 633064, "raw_text": "This is caused (in at least one case) by the fact that certdb_SaveSingleProfile returns SECFailure without calling PORT_SetError.\n\nI believe the rule is that any function exported by NSS must call PORT_SetError before returning SECFailure. I think there are probably many instances of this general problem and I'm not sure if each one should have its own bug report. I am just reporting this one as I ran across it while investigating other bugs.", "id": 5268417, "time": "2011-02-10T02:21:54Z", "attachment_id": null, "creator": "brian@briansmith.org", "author": "brian@briansmith.org", "text": "This is caused (in at least one case) by the fact that certdb_SaveSingleProfile returns SECFailure without calling PORT_SetError.\n\nI believe the rule is that any function exported by NSS must call PORT_SetError before returning SECFailure. I think there are probably many instances of this general problem and I'm not sure if each one should have its own bug report. I am just reporting this one as I ran across it while investigating other bugs.", "count": 0}, {"author": "nelson@bolyard.me", "text": "> This is caused (in at least one case) by the fact that \n> certdb_SaveSingleProfile returns SECFailure without calling PORT_SetError.\n\nWhat is \"This\" that is caused?", "count": 1, "attachment_id": null, "creator": "nelson@bolyard.me", "id": 5271910, "time": "2011-02-11T07:38:21Z", "is_private": false, "tags": [], "bug_id": 633064, "creation_time": "2011-02-11T07:38:21Z", "raw_text": "> This is caused (in at least one case) by the fact that \n> certdb_SaveSingleProfile returns SECFailure without calling PORT_SetError.\n\nWhat is \"This\" that is caused?"}]}}, "comments": {}}